src/components/form/components/FormDateTime.tsx

```diff
@@ -1,25 +1,16 @@
 import React from 'react';
-import { LocalizationProvider } from '@mui/x-date-pickers';
-import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
-import {
-  DateTimePicker,
-  DateTimePickerProps,
-} from '@mui/x-date-pickers/DateTimePicker';
+import * as Bootstrap from 'react-bootstrap';
 
-export interface FormDateTime extends DateTimePickerProps<Date> {}
+export interface FormDateTime extends Bootstrap.FormControlProps {}
```

Comment:
Prefer suffixing `Props` to prop types. I think you're already doing it with other prop types so must have only missed this one :)

```suggestion
export interface FormDateTimeProps extends Bootstrap.FormControlProps {}
```